### ISSUE TITLE
I have a few questions.

### ISSUE THREAD

**Issue by kwbyun** [McGregorWwww/UDTransNet#3]
1. Is it possible to learn by changing the model to UDTransNet and setting only the Custum data set?

2. When it is multi class, I wonder if semantic segmentation usually includes background learning. Please help I have nowhere to ask.

3. I want to know how to calculate the class weight. Is there any reference material?

4. Lastly, I wonder if you have published your thesis. It doesn't come out even if you search for it.

**Comment by McGregorWwww** [comment]
Thanks for your attention.

1. Certainly, if you have another pipeline, you can just copy the model code and set the correct input/output.
2. Yes, training with bg is a common way if you use softmax and one-hot encoding, however, you can also train without bg by using sigmoid and some binary losses.
3. You may refer to some class-imbalance methods, e.g., # CReST: A Class-Rebalancing Self-Training Framework for Imbalanced Semi-Supervised Learning.
4. The paper is just accepted by Medical Image Analysis, and will be published very soon.
